x-pack/platform/plugins/shared/task_manager/server/task.ts

```diff
@@ -255,6 +256,40 @@ export interface IntervalSchedule {
    * An interval in minutes (e.g. '5m'). If specified, this is a recurring task.
    * */
   interval: Interval;
+  rrule?: never;
+}
+
+export interface RruleSchedule {
+  rrule: RruleMonthly | RruleWeekly | RruleDaily;
+  interval?: never;
+}
+
+interface RruleCommon {
+  freq: Frequency;
+  interval: number;
+  tzid: string;
+}
+
+interface RruleMonthly extends RruleCommon {
+  freq: Frequency.MONTHLY;
+  bymonthday?: number[];
+  byhour?: number[];
+  byminute?: number[];
+  byweekday?: Weekday[];
+}
+interface RruleWeekly extends RruleCommon {
+  freq: Frequency.WEEKLY;
+  byweekday?: Weekday[];
+  byhour?: number[];
+  byminute?: number[];
+  bymonthday?: never;
+}
+interface RruleDaily extends RruleCommon {
```

Comment:
Seems like a common use case would be hourly, setting a particular minute, for instance, every hour at 10m, so it would go off at 1:10, then 2:10, etc.  Is that not supported?

Reply:
It is supported but, I discussed this with Mike, initially we start with daily+